### ISSUE TITLE
second section in blog entry ends up in right side pane

### ISSUE THREAD

**Issue by kbucheli** [preaction/Statocles#518]
I am using the master branch.
When dividing the blog entry into two sections, the template generation blows up:
```
    <body>
        <header>
...
        </header>
        <div class="main container">
            <div class="row">
                <div class="six columns">
$section[0]
                </div>
                <div class="six columns">
$section[1]
                </div>

            </div>
        </div>
        <footer>
...
        </footer>
    </body>
</html>
```
With one section only, I get as expected a lovely
```
    <body>
        <header>
....
        </header>
        <div class="main container">
            <div class="row">
                <div class="nine columns">
                    <main>
                        <header>
    <h1>$title</h1>
    <p><time datetime="2016-10-23">
        Posted on 2016-10-23
    </time>
        by $author
    <br> Tags:@tag_links
    </p>
</header>
<section id="section-1">
$section[1]
</section>
                    </main>
                </div>
                <div class="three columns sidebar">
                    
                        <nav id="tags">
        <h1>Tags</h1>
        <ul class="list-inline">
@global_tag_list
        </ul>
    </nav>                    
                </div>

            </div>
        </div>
        <footer>
...
        </footer>
    </body>
</html>
```
Somehow most of the template gets eaten...

**Comment by preaction** [comment]
Sorry, I'm not sure what the problem is. Could you post the entire template you're using, the data document (markdown file) you're using, and the expected output? If you have your site in a Github repo and can post a link, that will likely work too.

**Comment by kbucheli** [comment]
I found it: some old stuff I long forgot about when I played around with the layout.
Sorry, my fault.

**Comment by preaction** [comment]
No worries ;). The new sidebar/content sections thing (released in 0.075) caused the behavior you mentioned on a few of my sites, so it might've been that. I had to make sure to upgrade both Statocles and the theme to get it fixed.
